src/translator_ingest/ingests/bgee/download.yaml

```diff
@@ -0,0 +1,40 @@
+### This file is a YAML configuration that specifies
+### the data to be downloaded and ingested for this project.
+### The configuration is used by `kghub-downloader`.
+### For more information, see https://github.com/monarch-initiative/kghub-downloader
+---
+# For bgee
+
+- url: https://bgee.org/ftp/bgee_v15_0/download/calls/expr_calls/Homo_sapiens_expr_simple.tsv.gz
```

Comment:
Richard and Evan pointed me to some new functionality to get the download files by version pattern match when specified in the URL:

(see ctkp for details)